src/panels/media-browser/ha-panel-media-browser.ts

```diff
@@ -64,14 +65,21 @@ class PanelMediaBrowser extends LitElement {
                 "ui.components.media-browser.media-player-browser"
               )}
             </div>
-            <mwc-icon-button
-              .label=${this.hass.localize(
-                "ui.components.media-browser.choose-player"
-              )}
-              @click=${this._showSelectMediaPlayerDialog}
-            >
-              <ha-svg-icon .path=${mdiPlayNetwork}></ha-svg-icon>
-            </mwc-icon-button>
+            <div>
+              <mwc-icon-button
+                .label=${this.hass.localize(
+                  "ui.components.media-browser.choose-player"
+                )}
+                @click=${this._showSelectMediaPlayerDialog}
+              >
+                <ha-svg-icon .path=${mdiPlayNetwork}></ha-svg-icon>
+              </mwc-icon-button>
+              <paper-tooltip animation-delay="0" position="left">
+                ${this.hass.localize(
+                  "ui.components.media-browser.choose_player"
+                )}
+              </paper-tooltip>
```

Comment:
Let's not do this, a header button should not have a tooltip. It could have a title, and if it is not clear enough let's make it a regular button instead of an icon button.

Reply:
Changed to button with text